generator/composer.json

```diff
@@ -14,8 +14,7 @@
     },
     "require": {
         "php": ">=8.1",
-        "ext-mongodb": "^1.16.0",
-        "mongodb/builder": "@dev",
+        "ext-mongodb": "^1.17.0",
         "mongodb/mongodb": "^1.17.0@dev",
```

Comment:
Might as well hold this pull request for a few more minutes until 1.17.0 is tagged ^^

Reply:
Like this?
```suggestion
        "mongodb/mongodb": "^1.17.0",
```